pkg/plugins/models.go

```diff
@@ -197,3 +197,8 @@ func NewEnabledPlugins() EnabledPlugins {
 		Apps:        make([]*AppPlugin, 0),
 	}
 }
+
+type PluginErrorInfo struct {
```

Comment:
I would recommend fusing this type with `PluginError`, since there's no point in having both.

Reply:
Wasn't sure if there was a pattern to distinguish between internal model and DTO, but in this case they're quite simple and not the intended final result - so thanks for the suggestion 👍

Reply:
@wbrowne TBH I don't know either, just going off intuition :D